internal/route/routes.go

```diff
@@ -93,40 +95,29 @@ func HandleTodos(w http.ResponseWriter, r *http.Request) {
 
 // HandleTodoDone marks todo as done or undone
 func HandleTodoDone(w http.ResponseWriter, r *http.Request) {
-	if r.Method == "GET" {
+	if r.Method != "PUT" {
 		w.WriteHeader(http.StatusForbidden)
-		log.Fatalf("[error] Route only handles POST requests")
+		log.Fatalf("[error] Route only handles PUT requests")
 		return
 	}
 
-	b, err := ioutil.ReadAll(r.Body)
-	if err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		log.Fatalf("[error] failed to read request body: %v\n", err)
-	}
-
-	defer r.Body.Close()
-
-	todo := &model.Todo{}
-	err = json.Unmarshal(b, &todo)
-
-	if err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		log.Fatalf("[error] failed to unmarshal json: %v\n", err)
-	}
-
```

Comment:
So before we were parsing a bunch of JSON into a `Todo`. Instead we'll just use the URL which contains our `listId` and `todoId` and get it from the DB.